ifcb-parsing-processing/README.md

```diff
@@ -0,0 +1,117 @@
+# OOI CGSN IFCB Data Management Repo for IFCB Operators
+
+This repository contains Python scripts that can be used by IFCB operators to process raw IFCB header data for use in OOI IFCB logs and 
+OOI IFCB Dashboard Metadata input. 
+
+
+
+### Using the Jupyter Notebooks
+
+
+**IFCB_Log_Notebooks Organization**
+
+These notebooks contain...
+* IFCB_Quick_Operator_QC_Checks.ipynb (Verifies that .rois have corresponding .hdr files within a specified directory.)
+* IFCB_Log-Underway_IFCB_Data_Processing.ipynb (Creates a .csv of compiled IFCB data to be used to populate the OOI IFCB Log from IFCB underway .hdr files.)
+* IFCB_Log-Discrete_IFCB_Data_Processing.ipynb (Creates a .csv of compiled IFCB data to be used to populate the OOI IFCB Log from IFCB discrete .hdr files.)
+* Ship_Underway_Lat_and_Lon_Processing.ipynb (Processes and standardizes shipboard data from the R/V Armstrong so lat and lon can be added to underway IFCB data.)
+* IFCB_Dashboard_Metadata_CSV.ipynb (Creates metadata for the OOI IFCB Dashboard from the OOI IFBC Log Excel file.)
+
+* /IFCB_Logs (IFCB logs are manually uploaded to this folder; these logs are needed to run IFCB_Dashboard_Metadata_CSV.ipynb.)
+* /HDR_Summaries (Contains csvs output from the *_IFCB_Data_Processing.ipynbs; these files are used to create IFCB logs.)
+* /Merged_Ship_Underway_Files (Files created by Ship_Underway_Lat_and_Lon_Processing.ipynb + shipboard data; these files are used during the lat & lon processing portions of IFCB_Log-Underway_IFCB_Data_Processing.ipynb.)
+* /Metadata_CSVs (Contains files output from IFCB_Dashboard_Metadata_CSV.ipynb, which can be uploaded to the OOI IFCB Dashboard to associate filters and tags with Samples in the UI.) 
+
+**Installation**
+
+1) Clone the repo:
+
+`git clone [repo URL]`
+
+2) Set up a Python environment:
+
+`python -m venv venv
+source venv/bin/activate  # On macOS/Linux
+venv\Scripts\activate     # On Windows`
+
+3) Install dependencies:
+
+`pip install -r ifcb-parsing-processing/requirements.txt`
+
+4) Launch the Jupyter Notebooks (nativate to the IFCB_Log_Notebooks directory and launch Jupyter)
+
+`cd IFCB_Log_Notebooks
+jupyter notebook`
```

Comment:
These two commands appear on the same line in the compiled readme. This might be fixed by enclosing each command in its own set of backticks.